state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1000,6 +1024,31 @@ pub mod pallet {
 			Self::deposit_event(Event::<T>::UpdatedBuyInterval { buy_interval: new_buy_interval });
 			Ok(())
 		}
+
+		/// Updates the swap retry delay.
+		///
+		/// ## Events
+		///
+		/// - [UpdatedSwapRetryDelay](Event::UpdatedSwapRetryDelay)
+		///
+		/// ## Errors
+		///
+		/// - [BadOrigin](frame_system::BadOrigin)
+		/// - [ZeroSwapRetryDelayNotAllowed](pallet_cf_pools::Error::ZeroSwapRetryDelayNotAllowed)
+		#[pallet::call_index(13)]
+		#[pallet::weight(T::WeightInfo::update_buy_interval())]
+		pub fn update_swap_retry_delay(
```

Comment:
I don't think we need a new function here, we have an `update_pallet_config` function already - this seems like a good candidate to add there - and ofc add some test cases for it

Reply:
Great idea, did know it was a thing. I'll move the flip buy interval in there as well while im at it.

Reply:
Yeah sounds good